app/views/groups/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="all">
+  <div class="loginHeader groupHeader">
+    <%= link_to destroy_user_session_path, method: :delete, class: 'backArrow' do %>
+      <i class="icon-arrow-left"></i>
+    <% end %>
+    <h2 class="groupHeaderTitle">Categories</h2>
+    <div class="search">
+      <img src="https://i.postimg.cc/R0xX4NtN/pngfind-com-lupa-png-sin-fondo-4235556.png' border='0' alt='pngfind-com-lupa-png-sin-fondo-4235556'"/>
+    </div>
+  </div>
+
+  <h2 class="categories-title">
+      <span class="categories-text">Aggregate Spending</span>
+      <span class="categories-number">$<%= total_sum = @user.movements.sum(:amount) %></span>
+  </h2>
+  <div class="bodyGroup cat">
+    <div class="groupsContainer">
+      <% @groups.each do |group| %>
+        <div class="groupItem">
+          <%= link_to group_movements_path(group) do %>
```

Comment:
This is a mobile application so it would be nice to adjust some padding and margins so that the page content could fit the mobile screen just like it is in the original design. 

This is your design:

![Screenshot 2023-08-17 at 11 12 49 AM](https://github.com/federicaulzurrun/PennyLane/assets/79506105/12022ccf-ba3e-448e-ba20-a5f5b91e07d7)



Please kindly follow this design:
![Screenshot 2023-08-17 at 11 26 16 AM](https://github.com/federicaulzurrun/PennyLane/assets/79506105/d1287b63-1de3-42e5-961c-125a71d7803f)

Please Style your work and try as much as possible to match the given design. Always pay attention to detail.